lib/Parser.js

```diff
@@ -638,11 +641,14 @@ Parser.prototype.walkClassDeclaration = function walkClassDeclaration(statement)
 };
 
 Parser.prototype.walkSwitchCases = function walkSwitchCases(switchCases) {
-	switchCases.forEach(function(switchCase) {
-		if(switchCase.test)
+	for(var index = 0; index <= switchCases.length - 1; index++) {
```

Comment:
Yep a benchmarkCase with a complex AST, which uses these constructs. There will be a special message in benchmark log if you improve build time with a statistical confidence of 90%.